src/App.js

```diff
@@ -1,9 +1,32 @@
 import React from 'react';
+import styled, {createGlobalStyle} from 'styled-components';
+import Chatting from './components/Chatting';
```

Comment:
```suggestion
import chatting from './components/chatting';
```
파일 이름과 맞춰주셔야 에러가 안납니다!